components/homeComponents/HackCountdown.tsx

```diff
@@ -0,0 +1,209 @@
+      textAlign: 'center',
+      background: 'linear-gradient(90deg, #d4a6ff, #80baff)',
+      backgroundSize: '300%',
+      borderRadius: '30px',
+      zIndex: 1,
+      transition: 'background 1s',
+      marginTop: '40px',
+      userSelect: 'none',
+      outline: 'none',
+    },
+  };
+
+  return (
+    <>
+      <style>
+        {`
+          .button {
+            text-decoration: none;
+            position: relative;
+            border: none;
+            font-size: 18px;
+            font-family: inherit;
+            cursor: pointer;
+            color: #fff;
+            width: 218.55px;
+            height: 53.44px;
+            line-height: 53.44px;
+            text-align: center;
+            background: linear-gradient(90deg, #d4a6ff, #80baff);
+            background-size: 300%;
+            border-radius: 30px;
+            z-index: 1;
+            transition: background 1s;
+            margin-top: 40px;
+            user-select: none;
+            outline: none;
+          }
+          .button:hover {
+            animation: ani 8s linear infinite;
+            border: none;
+          }
+          .button::before {
+            content: "";
+            position: absolute;
+            top: -5px;
+            left: -5px;
+            right: -5px;
+            bottom: -5px;
+            z-index: -1;
+            background: linear-gradient(90deg, #d4a6ff, #80baff);
+            background-size: 400%;
+            border-radius: 35px;
+            transition: 1s;
+          }
+          .button:hover::before {
+            filter: blur(20px);
+          }
+          .button:focus {
+            outline: none;
+          }
+          @keyframes ani {
+            0% {
+              background-position: 0%;
+            }
+            100% {
+              background-position: 400%;
+            }
+          }
+        `}
+      </style>
+      <div style={styles.countdownContainer}>
+        <div style={styles.timeSection}>
+          {Object.entries(timeLeft).map(([unit, value]) => {
+            const digits = value.toString().padStart(2, '0').split('');
+            return (
+              <div key={unit} style={{ textAlign: 'center' }}>
+                <div style={{ display: 'flex' }}>
+                  {digits.map((digit, index) => (
+                    <div key={`${unit}-${index}`} style={digitBoxStyle}>
+                      <span style={digitStyle}>{digit}</span>
+                    </div>
+                  ))}
+                </div>
+                <div style={labelStyle}>{unit.toUpperCase()}</div>
+              </div>
+            );
+          })}
+        </div>
+        <div style={styles.notifySection}>
+          <p>We will let you know when we are launching</p>
+          <button className="button">Notify Me</button>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const Hack24Countdown: React.FC = () => {
+  return <HackCountdown targetDate="2024-11-04T00:00:00Z" />;
```

Comment:
i think it would be better to set targetDate as part of `hackportal.config.ts` and then import it and use as constant for the props